modules/common-attributes.adoc

```diff
@@ -9,6 +9,14 @@
 :prewrap!:
 :op-system-first: Red Hat Enterprise Linux CoreOS (RHCOS)
 :op-system: RHCOS
+:op-system-base: RHEL
+:op-system-base-full: Red Hat Enterprise Linux (RHEL)
+ifdef::openshift-origin[]
+:op-system-first: Fedora CoreOS (FCOS)
+:op-system: FCOS
+:op-system-base: Fedora
+:op-system-base-full: Fedora
+endif::[]
```

Comment:
@openshift/team-documentation The OKD team has created some new variables for our docs to represent different operating systems. Vikram wanted me to announce these changes to you before I merge. Let me know if you have any concerns.

Reply:
Only comment is to consider being consistent with the CoreOS attribute name by using `op-system-base-first` instead of `op-system-base-full`.

Reply:
@vrutkovs I didn't see this comment ^^ until after I merged. Must have been a refresh issue. Any thoughts?

Reply:
I'll include that in the next batch of updates